scripts/homepage.js

```diff
@@ -68,6 +68,11 @@ function process_payment (amount, token) {
     payment_http.send('amount=' + amount + '&token=' + token.id);
 }
 
+function payment_complete (amount) {
```

Comment:
I don't suppose it makes much difference, and it would remove the jquery.cookie dependency. Just be sure to set a far-future date (currently 10 years).